calc/calc.php

```diff
@@ -5,26 +5,28 @@
  * Version: 1.0
  * Author: Mike Macgirvin <http://macgirvin.com/profile/mike>
  */
+
+use Friendica\App;
 use Friendica\Core\Hook;
 use Friendica\DI;
 
 function calc_install() {
 	Hook::register('app_menu', 'addon/calc/calc.php', 'calc_app_menu');
 }
 
-function calc_app_menu($a,&$b) {
+function calc_app_menu(App $a, array &$b)
+{
 	$b['app_menu'][] = '<div class="app-title"><a href="calc">Calculator</a></div>'; 
 }
 
 
-function calc_module() {}
-
-
-
-
-function calc_init($a) {
+function calc_module()
+{
+}
```

Comment:
Please keep this on a single line, it's a statement rather than an actual function definition. The simple existence of this method is checked to figure out if the addon offers a module.
```suggestion
function calc_module() {}
```

Reply:
I copied your text almost unmodified in:
````
/**
 * Thi is a statement rather than an actual function definition. The simple
 * existence of this method is checked to figure out if the addon offers a
 * module.
 */
````